test/OpenTelemetry.Extensions.Propagators.Tests/OpenTelemetry.Extensions.Propagators.Tests.csproj

```diff
@@ -0,0 +1,32 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Unit test project for OpenTelemetry ASP.NET Core propagators</Description>
```

Comment:
the description is incorrect.. Do we need it for tests?

Reply:
I will remove it - while some test projects have it, others dont.